line-sdk/src/main/java/com/linecorp/linesdk/internal/IssueAccessTokenResult.java

```diff
@@ -20,12 +20,17 @@
     @NonNull
     private final List<Scope> scopes;
     @Nullable
+    private final String idTokenRawStr;
+    @Nullable
     private final LineIdToken idToken;
 
-    public IssueAccessTokenResult(
-            @NonNull InternalAccessToken accessToken, @NonNull List<Scope> scopes, @Nullable LineIdToken idToken) {
+    public IssueAccessTokenResult(@NonNull InternalAccessToken accessToken,
+                                  @NonNull List<Scope> scopes,
+                                  @Nullable final String idTokenRawStr,
+                                  @Nullable LineIdToken idToken) {
         this.accessToken = accessToken;
         this.scopes = Collections.unmodifiableList(scopes);
+        this.idTokenRawStr = idTokenRawStr;
```

Comment:
idTokenRawString is the raw string of LineIdToken, why not put it as a member of LineIdToken?

Reply:
I thought about this option at first, but I was not sure if it's a good idea to put `raw string` and `parsed fields` together

I will fix it now, if you think it's better

Reply:
fixed